docs/io/fits/appendix/faq.rst

```diff
@@ -1,5 +1,3 @@
-.. doctest-skip-all
```

Comment:
is the pyfits inherited appendix and history still needed, or maybe can be merged into the main docs somehow?

Reply:
Puh, that's a tricky one. I probably convoluted too many fixes in the PR. I originally wanted to change only some documentation, then started working on making the docs pass and then I changed all (possible) links to https...

I'll close the PR now anyway because I just wanted to see if it worked for the full CI.